python/lsst/ap/association/packageAlerts.py

```diff
@@ -63,6 +77,12 @@ class PackageAlertsConfig(pexConfig.Config):
         default=os.path.join(os.getcwd(), "alerts"),
     )
 
+    produceAlertsConfig = pexConfig.Field(
```

Comment:
```suggestion
    doProduceAlerts = pexConfig.Field(
```

Reply:
Swapped to this and added `doWriteAlerts`